arm-batch/2015-12-01/BatchManagement.json

```diff
@@ -0,0 +1,1340 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BatchManagement",
+    "version": "2015-12-01",
+    "x-ms-code-generation-settings": {
+      "name": "BatchManagementClient"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Batch/batchAccounts/{accountName}": {
+      "put": {
+        "tags": [
+          "Account"
+        ],
+        "operationId": "Account_Create",
+        "description": "Begin creating the batch account.To determine whether the operation has finished processing the request, call GetLongRunningOperationStatus.",
```

Comment:
Is the definition of "GetLongRunningOperationStatus" in the client runtime for all the languages? This was required for hydra generated clients. AutoRest generated clients have inbuilt polling for long running operations.